plugins/com.google.cloud.tools.eclipse.appengine.validation.test/META-INF/MANIFEST.MF

```diff
@@ -8,7 +8,8 @@ Bundle-Vendor: Google Inc.
 Bundle-Version: 0.1.0.qualifier
 Fragment-Host: com.google.cloud.tools.eclipse.appengine.validation
 Require-Bundle: org.junit,
- org.eclipse.equinox.registry
+ org.eclipse.equinox.registry,
+ com.google.cloud.tools.eclipse.test.dependencies
```

Comment:
My current thinking is that we should prefer Import-Package to Require-Bundle. Import-Package is the OSGi standard. Require-Bundle is an Eclipse extension that seems lightly deprecated.